x/lightclient/ante/ibc_msg_update_client.go

```diff
@@ -62,7 +62,17 @@ func (i IBCMessagesDecorator) HandleMsgUpdateClient(ctx sdk.Context, msg *ibccli
 		return gerrc.ErrInternal.Wrapf("get rollapp from sequencer: rollapp: %s", seq.RollappId)
 	}
 
-	// TODO: in hard fork will need to also use revision to make sure not from old revision
+	// cannot update the LC unless fork is resolved (after receiving state updates of HF height +1 & HF height +2
+	if i.k.IsHardForkingInProgress(ctx, rollapp.RollappId) {
+		return types.ErrorHardForkInProgress
+	}
```

Comment:
not needed if client is frozen

Reply:
I might leave for explicity

Reply:
I guess it also prevents gov based revival of the client